### PR TITLE
Tensor persist

### PR DESCRIPTION
**Pull request by jlebensold**
This PR introduces postgres and a few SQLAlchemy models for storing workers and tensors. Next step would be hooking it up to the rest_api.

### CODE REVIEW

**Comment by mari-linhares** [comment]
@jlebensold feel free to create a few issues related to these changes!

**Comment by jlebensold** [comment]
@mari-linhares please let me know if you can give this a gander :)

**Comment by mari-linhares** [comment]
I'm on it @jlebensold

File: client_routines.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import torch as th
```
Comment (mari-linhares) [inline client_routines.py]: Is this file really needed?

File: grid/app/models.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+from . import db
```
Comment (mari-linhares) [inline grid/app/models.py]: from . import db -> from grid.app.models import db

File: grid/app/models.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+from sqlalchemy.orm import sessionmaker, relationship
```
Comment (mari-linhares) [inline grid/app/models.py]: make this a multiple lines import

File: grid/app/models.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+
+class Worker(db.Model):
+    __tablename__ = "workers"
+
+    id = db.Column(db.Integer, primary_key=True)
+    public_id = db.Column(db.String(64), index=True, unique=True)
+    worker_objects = db.relationship("WorkerObject", backref="workers", lazy="dynamic")
+
+    def tensor(self):
```
Comment (mari-linhares) [inline grid/app/models.py]: maybe use @property?

File: grid/example.py [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# from .app import db
```
Comment (mari-linhares) [inline grid/example.py]: Remove this file?

File: grid/app/models.py [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+import syft as sy
+
+
+class Worker(db.Model):
+    __tablename__ = "workers"
+
+    id = db.Column(db.Integer, primary_key=True)
+    public_id = db.Column(db.String(64), index=True, unique=True)
+    worker_objects = db.relationship("WorkerObject", backref="workers", lazy="dynamic")
```
Comment (mari-linhares) [inline grid/app/models.py]: Some comments could help :)

File: grid/app/models.py [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+        return f"<Tensor {self.id}>"
+
+
+class WorkerObject(db.Model):
+    __tablename__ = "worker_objects"
+    id = db.Column(db.Integer, primary_key=True)
+    description = db.Column(db.String(64), index=True, unique=True)
+    data = db.Column(db.Binary(64))
+    worker_id = db.Column(db.Integer, db.ForeignKey("workers.id"))
+    worker = db.relationship("Worker", load_on_pending=True)
```
Comment (mari-linhares) [inline grid/app/models.py]: Some comments could help :)

File: grid/migrations/README [hunk 8]
```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```
Comment (mari-linhares) [inline grid/migrations/README]: More details could help :)

File: grid/migrations/versions/b2ef8dc4f38b_workers_table.py [hunk 9]
```diff
@@ -0,0 +1,49 @@
+"""workers table
```
Comment (mari-linhares) [inline grid/migrations/versions/b2ef8dc4f38b_workers_table.py]: do we want/ to keep this file?

File: test/test_flask_integration.py [hunk 10]
```diff
@@ -0,0 +1,9 @@
+import os
```
Comment (mari-linhares) [inline test/test_flask_integration.py]: ?

File: test/test_models.py [hunk 11]
```diff
@@ -0,0 +1,46 @@
+from grid.app.models import Worker, WorkerObject
```
Comment (mari-linhares) [inline test/test_models.py]: make this a multiple lines import

File: grid/migrations/versions/b2ef8dc4f38b_workers_table.py (hunk 9 above)
Comment (jlebensold) [reply grid/migrations/versions/b2ef8dc4f38b_workers_table.py]: yes absolutely. This defines the schema for our persistence layer
